es.md

```diff
@@ -11,5 +11,5 @@ katuma_description: "[Katuma](http://www.katuma.org) es una plataforma cooperati
 
 timeoverflow_description: "[Time Overflow](http://timeoverflow.org) es una plataforma de gestión de Bancos de Tiempo, un sistema de oferta de servicios a cambio de tiempo. La plataforma facilita las tareas de gestión e intercambio de servicios entre miembros de un banco."
 
-friends_title: "Colaboradores"
+friends_title: "Colaboramos con:"
```

Comment:
I did the opposite change in https://github.com/coopdevs/coopdevs.github.io/pull/18 to be consistent with the `Proyectos` title, which is a single noun without `:`. What do you guys think? @enricostano @danypr92 @naoisegolden

Reply:
I think both are correct, what we need to decide is the tone. "Colaboradores" is standard, "Colaboramos con" is more personal, carefree, down-to-earth. I think our tone is more "Colaboramos con". Also, "Colaboradores" tens to be a relationship where them are collaborating with us, whereas "Colaboramos con" is more a equal relationship. That said, "Colaboradores" is ok with me too, if that is what we are using elsewhere.

About the colon (":"), I think it is grammatically correct to use it with "Colaboramos con", but perhaps it's not visually nice. I have no strong opinion. If "Colaboradores" definitely no colon.

Reply:
I tried to express what @naoisegolden point to "Colaboramos con" is more a equal relationship
Not all organizations collaborate with us, in some we collaborate with them :nerd_face:

Reply:
remove : is good to me

Reply:
Ok, so "Colaboramos como" but without the `:` then?